src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -67,6 +67,8 @@ export type OpportunityMetadataBase = {
   underlyingAssetIds: AssetIdsTuple
   // The underlying amount of underlyingAssetId 0 and maybe 1 per 1 LP token, in base unit
   underlyingAssetRatiosBaseUnit: readonly string[]
+  // The underlying balancing ratios of every asset
+  underlyingAssetRatios?: readonly string[]
```

Comment:
Can we use `underlyingAssetRatiosBaseUnit` here, i.e how does that differ from it?

Reply:
It differs from it, `underlyingAssetRatiosBaseUnit` gives the amount of asset you expect to have in the pool against 1 unit of base asset while `underlyingAssetRatios` are the balancing ratios of every underlying assets from 0 to 1, if you add every underlying asset ratios it will ends up being 1, permits to show every balancing percentages of every pools

Reply:
Let's discuss this some more - this is effectively the same as we have previously and we should be able to implement this without implementing a new property here

Reply:
Discussed this one out and actually LFG